components/header/_header2.scss

```diff
@@ -0,0 +1,65 @@
+$header2-bp: 36em;
+
+.header2 {
+  display: block;
+  overflow: hidden;
+  padding: 1.4rem 0 1rem;
+  &::before {
+    content: "";
+    display: block;
+    width: 100%;
+    height: 8px;
+    background-color: $color-secondary;
+    position: absolute;
+    top: 0;
+    left: 0;
+    margin: 0;
+  }
+  @include mq($header2-bp) {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.header2__logo {
+  margin-bottom: 1rem;
+  @include mq($header2-bp) {
+    margin-bottom: 0;
+    flex: 1 1 100%;
+  }
+}
+
+.header2__info {
+  white-space: nowrap;
+  min-height: 3rem;
+  width: 550px;
```

Comment:
What's the reason for using `px` here?

Reply:
I can't find any references to this header element in eq and ras frontstage--do you know if we still need this?

Reply:
Looking at this in the browser, not sure why we need a width at all.

Reply:
Decision to remove this element after talks with @armstrongb. Is no longer used